bouncer.go

```diff
@@ -279,8 +283,12 @@ func (bouncer *Bouncer) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
 
 	// Right here if we cannot join the stream we forbid the request to go on.
 	if bouncer.crowdsecMode == configuration.StreamMode || bouncer.crowdsecMode == configuration.AloneMode {
+		//nolint:gocritic
 		if isCrowdsecStreamHealthy {
 			handleNextServeHTTP(bouncer, remoteIP, rw, req)
+		} else if bouncer.nbFailedStreamUpdate < bouncer.maxFailedStreamUpdate || bouncer.maxFailedStreamUpdate == -1 {
+			bouncer.nbFailedStreamUpdate++
```

Comment:
If the `maxFailedStreamUpdate` is set to 2 and 3 people try to access a route, you block the third and all the next ones

Reply:
That's true, we need to increment only once every stream update